src/felles/komponenter/rekbis-kort/RekBisKort.tsx

```diff
@@ -0,0 +1,23 @@
+import { Panel } from '@navikt/ds-react';
+import classNames from 'classnames';
+import * as React from 'react';
+import css from './RekBisKort.module.css';
+
+export interface IRekBisKort {
```

Comment:
Vi har ingen konvensjon for å prefikse interfaces med `I` – kan være forvirrende for andre utviklere på teamet at konvensjonen plutselig dukker opp her?